mu/app.py

```diff
@@ -256,6 +258,36 @@ def setup_modes(editor, view):
     }
 
 
+class MutexError(BaseException):
+    pass
+
+
+class SharedMemory(object):
```

Comment:
The name of this class might imply this is a generic way to share memory, should we maybe rename to something more indicative to this single instance lock mechanism?